test/tests/kernels/hfem/dirichlet.i

```diff
@@ -40,18 +42,32 @@
 
 [Kernels]
   [diff]
-    type = ArrayDiffusion
+    type = MatDiffusion
     variable = u
+    diffusivity = '1'
     block = 0
-    diffusion_coefficient = dc
   []
   [source]
-    type = ArrayCoupledForce
+    type = CoupledForce
     variable = u
     v = v
-    coef = '1 2'
+    coef = '1'
     block = 0
   []
+  [reaction]
+    type = Reaction
+    variable = uhat
+    rate = '1'
+    block = BOUNDARY_SIDE_LOWERD_SUBDOMAIN
+    reaction_coefficient = rc
```

Comment:
this parameter is not valid for Reaction?

Reply:
Oh. I deleted